library/fixbook/FixBookUI.java

```diff
@@ -48,9 +48,9 @@ public void runFixBookMethods() {
 				break;	
 				
 			case FIXING:
-				String AnS = input("Fix Book? (Y/N) : ");
+				String anSwer = input("Fix Book? (Y/N) : ");

```

Comment:
"anSwer" should be "answer"

Reply:
@BLakshmiPrasanna Rework needed for particular error raised by Haris Ali

Reply:
@abhimaniac4 Already changed this issue in commit [d0a3ff2] please review